app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>All Contestants:</h1>
+<ul>
+  <% @contestants.each do |contestant| %>
+    <div id="contestant-<%= contestant.id %>">
+      <li><%= contestant.name %></li>
+      <li>Projects: <%= contestant.names_of_projects.join(", ") %></li>
```

Comment:
Nice work making the comma separated list! And as it is display logic, it's great for the view